src/components/Article.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import getArticles from "../../api";
+
+export default function Articles() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles()
+        .then((articlesData) => {
+            setArticles(articlesData.sortedArticles);
+        })
+    }, [])
+
+    return (
+        <>
+        {articles.map((article) => {
+            console.log(article.article_img_url)
+            return (
+                <div className="article-card">
+                    <h1>{article.title}</h1>
```

Comment:
consider what other info the user might want to see about each article before they decide which on they'd like to click on, e.g. We also want to see when the articles were created in a human readable format as they should already be sorted by date when they come back from the api as well as comment count and votes, image etc